pkg/manifestreader/common.go

```diff
@@ -87,12 +81,11 @@ func SetNamespaces(factory util.Factory, infos []*resource.Info,
 		// namespace-scoped. If it is the latter, we set the namespace
 		// to the provided default.
 		var scope string
-		for _, crdInf := range crdInfos {
-			u, _ := crdInf.Object.(*unstructured.Unstructured)
-			group, _, _ := unstructured.NestedString(u.Object, "spec", "group")
-			kind, _, _ := unstructured.NestedString(u.Object, "spec", "names", "kind")
+		for _, crdObj := range crdObjs {
+			group, _, _ := unstructured.NestedString(crdObj.Object, "spec", "group")
+			kind, _, _ := unstructured.NestedString(crdObj.Object, "spec", "names", "kind")
```

Comment:
Better to use `unstructured.GetKind()` here.

Reply:
Same as above.